charts/gsp-cluster/templates/02-gsp-system/service-operator/role.yaml

```diff
@@ -26,14 +27,22 @@ rules:
   - patch
   - update
 - apiGroups:
-  - ""
+  - queue.gsp.k8s.io
   resources:
-  - secrets
```

Comment:
Do we want secrets here? Doesn't the operator [store credentials in secrets](https://github.com/alphagov/gsp/pull/381/files#diff-a32e0cd8964165cde558f9fff5eaecffR48-R49) so needs access to them? Or is there another mechanism for that?